sources/academy/platform/get_most_of_actors/monetizing_your_actor.md

```diff
@@ -5,169 +5,130 @@ sidebar_position: 5
+</details>
 
 ## Pay-per-result pricing model
 
-If you make your Actor pay-per-result instead, you will set the price per 1,000 results, and users will be charged for your Actor solely based on the number of results your Actor produces. That's your _revenue_. The underlying [platform usage](https://docs.apify.com/platform/actors/running/usage-and-resources) the Actor generates, using the pay-per-result unit pricing defined below, is your _cost_. Your profit which is then paid to you is computed as 80% of the revenue minus the costs, i.e. `(0.8 * revenue) - costs = profit`. This profit is also paid out monthly.
+In this model, you set a price per 1000 results. Users are charged based on the number of results your Actor produces. Your profit is calculated as 80% of the revenue minus platform usage costs. The formula is:
+
+`(0.8 * revenue) - costs = profit`
 
 ### Pay-per-result unit pricing for cost computation
 
-| Service                         | Unit price                 |
-|---------------------------------|----------------------------|
-| Compute unit                    | **$0.4** / CU              |
-| Residential proxies             | **$13** / GB               |
-| SERPs proxy                     | **$3** / 1,000 SERPs       |
-| Data transfer - external        | **$0.20** / GB             |
-| Data transfer - internal        | **$0.05** / GB             |
-| Dataset - timed storage         | **$1.00** / 1,000 GB-hours |
-| Dataset - reads                 | **$0.0004** / 1,000 reads  |
-| Dataset - writes                | **$0.005** / 1,000 writes  |
+| Service | Unit price |
+|:---|:---|
+| Compute unit | **$0.4** / CU |
+| Residential proxies | **$13** / GB |
+| SERPs proxy | **$3** / 1,000 SERPs |
+| Data transfer - external | **$0.20** / GB |
+| Data transfer - internal | **$0.05** / GB |
+| Dataset - timed storage | **$1.00** / 1,000 GB-hours |
+| Dataset - reads | **$0.0004** / 1,000 reads |
+| Dataset - writes | **$0.005** / 1,000 writes |
 | Key-value store - timed storage | **$1.00** / 1,000 GB-hours |
-| Key-value store - reads         | **$0.005** / 1,000 reads   |
-| Key-value store - writes        | **$0.05** / 1,000 writes   |
-| Key-value store - lists         | **$0.05** / 1,000 lists    |
-| Request queue - timed storage   | **$4.00** / 1,000 GB-hours |
-| Request queue - reads           | **$0.004** / 1,000 reads   |
-| Request queue - writes          | **$0.02** / 1,000 writes   |
+| Key-value store - reads | **$0.005** / 1,000 reads |
+| Key-value store - writes | **$0.05** / 1,000 writes |
+| Key-value store - lists | **$0.05** / 1,000 lists |
+| Request queue - timed storage | **$4.00** / 1,000 GB-hours |
+| Request queue - reads | **$0.004** / 1,000 reads |
+| Request queue - writes | **$0.02** / 1,000 writes |
```

Comment:
Nit: I find this harder to read in source code

Reply:
I added more spacing to make it more readable in the source